src/lib/operators/abstract_aggregate_operator.hpp

```diff
@@ -0,0 +1,119 @@
+#pragma once
+
+#include "expression/aggregate_expression.hpp"
+#include "operators/abstract_operator.hpp"
+#include "operators/abstract_read_only_operator.hpp"
+#include "type_comparison.hpp"
+#include "types.hpp"
+
+namespace opossum {
+
+/**
+ * Aggregates are defined by the column (ColumnID for Operators, LQPColumnReference in LQP) they operate on and the aggregate
+ * function they use. COUNT() is the exception that doesn't use a column, which is why column is optional
```

Comment:
`COUNT() is the exception that does **not necessarily** use a column`. For example, `COUNT(a)` would check how many **non-NULL** values are in a.

`OperatorsAggregateTest.CanCountStringColumns` should cover that.

Reply:
should already be updated, I believe the code preview from github is just outdated